crates/base_db/src/input.rs

```diff
@@ -169,7 +171,7 @@ pub struct ProcMacro {
 impl Eq for ProcMacro {}
 impl PartialEq for ProcMacro {
     fn eq(&self, other: &ProcMacro) -> bool {
-        self.name == other.name && Arc::ptr_eq(&self.expander, &other.expander)
+        self.name == other.name && self.expander.get_ptr() == other.expander.get_ptr()
```

Comment:
The method can be inlined here

```suggestion
        self.name == other.name && Arc::as_ptr(&self.expander) as *const () == Arc::as_ptr(&other.expander) as *const ()
```